stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/feature/messages/list/options/message/MessageOptionsDialogFragment.kt

```diff
@@ -448,7 +441,6 @@ public class MessageOptionsDialogFragment : FullScreenDialogFragment() {
                 it.message = message
                 it.optionsDialogType = optionsDialogType
                 it.style = style
-                it.attachmentFactoryManager = attachmentFactoryManager
```

Comment:
`attachmentFactoryManager` is already passed to `messageListItemViewHolderFactory` above, so we don't need to keep a reference to `attachmentFactoryManager` itself, cause we can use `messageListItemViewHolderFactory` instead